cli/cmd/generate.go

```diff
@@ -196,7 +196,7 @@ func determineOutputDirPath(location string) (string, error) {
 }
 
 // writeHclOutputPreCheck Prompt for confirmation if main.tf already exists; return true to continue
-func writeHclOutputPreCheck(outputLocation string) (bool, error) {
+func writeHclOutputPreCheck(outputLocation string, cloud string) (bool, error) {
```

Comment:
nit: do we what to call this cloud? I believe on the call yesterday we discussed some examples of TF that wasn't creating cloud infra.